templates/wechatprogram/miniapp-adapter.js

```diff
@@ -52,6 +52,39 @@ exports.adapt = function (canvas, width, height) {
         };
     };
 
+    wx.onKeyboardInput = function (cb) {
+        GameGlobal._onKeyboardInput = function (event) {
+            let res = event.detail;
+            cb(res);
+        };
+    };
+
+    wx.onKeyboardConfirm = function (cb) {
+        GameGlobal._onKeyboardConfirm = function (event) {
+            let res = event.detail;
+            cb(res);
+        };
+    };
+
+    wx.onKeyboardComplete = function (cb) {
+        GameGlobal._onKeyboardComplete = function (event) {
+            GameGlobal.indexThis.setData({
+                showInput: false,
+            });
+            let res = event.detail;
+            cb(res);
```

Comment:
```suggestion
            cb(event.detail);
```

Reply:
the same as onKeyboardConfirm   onKeyboardInput

Reply:
done.